opm/parser/eclipse/EclipseState/Schedule/Group.hpp

```diff
@@ -51,6 +51,7 @@ namespace Opm {
         bool isProductionGroup(size_t timeStep) const;
         bool isInjectionGroup(size_t timeStep) const;
         void setProductionGroup(size_t timeStep, bool isProductionGroup);
+        void setInjectionGroup(size_t timeStep, bool isInjectionGroup_);
```

Comment:
In my understanding, a group that contains injection wells and production wells at the same time provides some mechanism to let injection wells and production wells interact with each other, such like making the injection rate for some phase dependent on the production rate.